test/action_controller/json_api/pagination_test.rb

```diff
@@ -0,0 +1,116 @@
+require 'test_helper'
+require 'will_paginate/array'
```

Comment:
Might we want to use the WillPaginate interface that doesn't monkey-patch Array in our tests?  I imagine this could cause bugs.